### DISCUSSION TITLE
q-uploader Reloading the Page on Upload

### DISCUSSION THREAD

**Discussion by fhf-sc** [quasarframework/quasar#12148] [General - Components / Directives / etc]
I am successfully using the q-uploader component; however, it seems to do a form submit and reload the page.  How can I prevent this default behavior?

`                <q-uploader
                  label="Upload truck photos"
                  flat
                  :url="uploadUrl"
                  :form-fields="[{name:'trucknum', value:selTruck.TruckNum}]"
                  style="max-width: 100%; max-height:100%;"
                  accept=".jpg, .jpeg"
                  max-files="10"
                  auto-upload
                  multiple
                  batch
                  @added="fileAdded"
                  @uploaded="fileUploaded"
                />
`

**Comment by yusufkandemir** [comment]
QUploader component doesn't have such behavior. You might have some logic that does that inside `fileUploaded` function or somewhere else. So, if you can't find it, or still think it is related to QUploader, then please share a repro so we can take a look.
